### PR TITLE
allow skipping multiple labels, rename ElideCheck to Skip

### PR DESCRIPTION
**Pull request by simeonschaub**
This still needs a nicer API, but I think these changes would be quite useful.

### CODE REVIEW

**Comment by simeonschaub** [comment]
@jw3126 Do you have any preferences about how the syntax for multiple labels should look like? I'm not sure which of `@skip a b c f(x)`, `@skip [a,b,c] f(x)` or `@skip (a,b,c) f(x)` would be more intuitive. Or maybe just leave this at the lower level API for now?

**Comment by jw3126** [comment]
I think I would go with `[a,b,c]` or `(a,b,c)`. It makes it easier to add other arguments to `@skip` later on.

File: src/OptionalArgChecks.jl [hunk 1]
```diff
@@ -44,32 +44,34 @@ macro mark(label, ex)
...
+    Skip(labels::Symbol...) = new{labels}()
 end
 
-@dynamo function (::ElideCheck{label})(x...) where {label}
+@dynamo function (::Skip{labels})(x...) where {labels}
     ir = IR(x...)
     ir === nothing && return
-    depth = 0
+
+    tape = ()
```
Comment (jw3126) [inline src/OptionalArgChecks.jl:55]: Nothing wrong with using a `Tuple`. I am just curious, why don't you `push!/pop!` to a `Vector` for this?

Comment (simeonschaub) [reply src/OptionalArgChecks.jl:55]: No reason really. Just that labels was a tuple, and that's what first came to mind

**Comment by simeonschaub** [comment]
Ok, I went with the `[a, b, c]` syntax now

**Comment by simeonschaub** [comment]
It's merged now, but somehow GitHub doesn't realize that automatically

**Pull request closed without merging**